lightningnetwork/lnd
# lnrpc+itest: return channel Memo for Pending channels

## Change Description
In a previous PR #7668  we added a Memo field for channels that could be specified when opening a channel. This was a reference note-to-self with no bearing on the functioning of the channel. In that PR, the memo value was returned only through ListChannels. This PR builds upon that PR by also returning the Memo field for pending open and waiting close channels.

## Steps to Test
An itest has been updated to verify that the Memo is returned but to manually test, one can verify the Memo field on a pending-open or waiting-close channel

## Pull Request Checklist
### Testing

### Code Style and Documentation

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.
